<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="5">
         <widget class="QLabel" name="registro_label">
          <property name="text">
           <string>Registro</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="4" colspan="4">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="disponibilidad_pushButton">
          <property name="text">
           <string>Comprobar disponibilidad</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="4">
            <widget class="QLabel" name="origen_label">
             <property name="text">
              <string>Origen</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="4">
            <widget class="QLabel" name="direccion_label">
             <property name="text">
              <string>Direcci√≥n</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="velocidad_label">
             <property name="text">
              <string>Velocidad</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="direccion_y_label">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="origen_y_label">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="origen_x_label">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QSpinBox" name="origen_x_spinBox"/>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QSpinBox" name="origen_y_spinBox"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="direccion_x_label">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="3">
            <widget class="QSpinBox" name="direccion_x_spinBox"/>
           </item>
           <item row="5" column="1" colspan="3">
            <widget class="QSpinBox" name="direccion_y_spinBox"/>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QSlider" name="velocidad_horizontalSlider">
             <property name="maximum">
              <number>150</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QLabel" name="velocidad_contador_label">
             <property name="text">
              <string>000</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" rowspan="2">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="color_label">
             <property name="text">
              <string>Color</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="red_label">
             <property name="text">
              <string>Red</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="red_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="green_label">
             <property name="text">
              <string>Green</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="green_spinBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="blue_label">
             <property name="text">
              <string>Blue</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="blue_spinBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="limpiar_pushButton">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="imprimir_pushButton">
          <property name="text">
           <string>Imprimir</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="agregar_inicio_pushButton">
          <property name="text">
           <string>Agregar Inicio</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="agregar_final_pushButton">
          <property name="text">
           <string>Agregar Final</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="id_lineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>Tabla de busqueda</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="tabla_de_busqueda_label">
          <property name="text">
           <string>Tabla de busqueda</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QPushButton" name="buscar_tabla_pushButton">
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="buscar_tabla_lineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="2">
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="imprimir_tabla_pushButton">
             <property name="text">
              <string>Imprimir Tabla</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="limpiar_tabla_pushButton">
             <property name="text">
              <string>Limpiar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>P√°gina</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="4">
         <widget class="QPushButton" name="imprimir_particula_scene">
          <property name="text">
           <string>Imprimir Particula</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QGraphicsView" name="scene"/>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLineEdit" name="buscar_scene_lineEdit"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="imprimir_todas_scene">
          <property name="text">
           <string>Imprimir Todas</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QPushButton" name="limpiar_scene">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar (ctrl + s)</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir (ctrl + o)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
